src/target/riscv/riscv-013.c

```diff
@@ -3685,15 +3759,18 @@ static int write_memory_abstract(struct target *target, target_addr_t address,
 			}
 		}
 
+		/* TODO:
+		 * (1) Only the 1st access can result in a 'skip'
+		 * (2) Analyze cmderr value */
 		if (result != ERROR_OK)
-			return result;
+			return MEM_ACCESS_STATUS_SKIPPED_ABSTRACT_ACCESS_CMDERR;
 
 		if (info->has_aampostincrement == YNM_YES)
 			updateaddr = false;
 		p += size;
 	}
 
-	return result;
+	return (result == ERROR_OK) ? MEM_ACCESS_STATUS_OK : MEM_ACCESS_STATUS_FAILED;
```

Comment:
(same as above)

It looks that the check for `result == ERROR_OK` is redundant here.

Reply:
Addressed